demos/storyboard/src/app.ts

```diff
@@ -100,11 +105,11 @@ export class App extends Border2 {
         if (!samples[defaultSelection]) {
             if (defaultSelection[0] === "@") {
                 // @lzxue/the-world-grid-map-use-l7
-                inspect(defaultSelection).then(md => {
-                    samples[defaultSelection] = md;
-                });
+                // inspect(defaultSelection).then(md => {
+                //     samples[defaultSelection] = md;
+                // });
```

Comment:
Was this intentional?

Reply:
Yes - I needed to do some quick fixes to the App to get the PR to compile!